compiler/z/codegen/OMRInstOpCode.hpp

```diff
@@ -724,6 +724,16 @@ class InstOpCode: public OMR::InstOpCode
    uint8_t getInstructionFormat() { return getInstructionFormat(_mnemonic);}
    const uint8_t * getOpCodeBinaryRepresentation() { return getOpCodeBinaryRepresentation(_mnemonic);}
 
+   /** \brief
+     *    Gets the 20-bit displacement mnemonic which is equivalent to a 12-bit displacement mnemonic.
+     *
+     *  \param op
+     *     The 12-bit displacement mnemonic for which to find an equivalent 20-bit displacement mnemonic.
+     *
+     *  \return
+     *     The 20-bit displacement mnemonic if it exists, otherwise `BAD` mnemonic.
+     */
+   static Mnemonic getLongDisplacementMnemonic(Mnemonic op);
```

Comment:
Small nit but do you think `getEquivalentLongDisplacementMnemonic` or `getLongDisplacementEquivalent` would be a better name here? I think it makes it a little more clear to a first time reader what the routine intends to do.

Reply:
I agree. Fixed in 66fe5c9